lib/service/Service.ts

```diff
@@ -258,14 +264,21 @@ class Service {
     argChecks.POSITIVE_AMOUNT({ amount });
     argChecks.VALID_CURRENCY({ currency });
     try {
-      const nodePubKey = isNodePubKey(args.nodeIdentifier) ? args.nodeIdentifier : this.pool.resolveAlias(args.nodeIdentifier);
-      const peer = this.pool.getPeer(nodePubKey);
-      await this.swapClientManager.openChannel({
-        peer,
-        amount,
-        currency,
-        pushAmount,
-      });
+      if (nodeIdentifier === 'Connext') {
```

Comment:
I'm not convinced this is a better solution. 

Also, a bit foggy on how the implementation details will look like. The peer has to be resolved on the service level since SwapClientManager has no access to Pool (imho, that's a good thing). Let's sync and talk this through offline.

Reply:
Current approach looks great, exactly what I had in mind.